src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let value = 0;
+  const payPerDay = 40;
+  const percentThreeDay = 20;
+  const percentSevenDay = 50;
+
+  if (days >= 3 && days < 7) {
+    value += (days * payPerDay) - percentThreeDay;
+  } else if (days >= 7) {
+    value += (days * payPerDay) - percentSevenDay;
+  } else {
+    value += days * payPerDay;
+  }
```

Comment:
you use payPerDay 3 times here
Would be better to make it a separate variable

And you can skip declaring 'value' variable, you may return it from each if
```suggestion
  if (days >= 3 && days < 7) {
    return (days * payPerDay) - percentThreeDay;
  } 
  
  if (days >= 7) {
    return (days * payPerDay) - percentSevenDay;
  } 
    
  return value += days * payPerDay;
  
```